src/org/authInfo.ts

```diff
@@ -813,7 +814,16 @@ export class AuthInfo extends AsyncOptionalCreatable<AuthInfo.Options> {
     }
     // If a username with NO oauth options, ensure authorization already exist.
     else if (username && !authOptions && !(await this.stateAggregator.orgs.exists(username))) {
-      throw messages.createError('namedOrgNotFound', [username]);
+      throw SfError.create({
+        name: 'NamedOrgNotFoundError',
+        message: messages.getMessage('namedOrgNotFound', [username]),
+        actions: [
+          `It looks like you mistyped the username or alias. Did you mean "${findSuggestion(username, [
```

Comment:
Could it be misleading to tell the user that they mistyped the username or alias? What happens if they provide a username or alias that simply doesn't exist? 

Maybe simplifying it to just `Did you mean "<suggestion>"?` would be the most flexible

Also do we need to remove the action if no suggestion is found?

Reply:
Juliet and I had some back and forth on this. We thought the current messaging was friendlier, but we also had exactly what you suggested 🤷‍♂️ 

I'll remove it if nothing is passed, or we couldn't deduce any similar names

Reply:
I'll happily defer to Juliet on the message 👍